cardano-cli/src/Cardano/CLI/EraBased/Run/Governance/Actions.hs

```diff
@@ -119,29 +151,43 @@ runGovernanceActionCreateConstitutionCmd cOn (NewConstitutionCmd network deposit
     constitutionHashSourceToHash constitutionHashSource
       & firstExceptT GovernanceActionsCmdConstitutionError
 
-  let prevGovActId = Ledger.maybeToStrictMaybe $ uncurry createPreviousGovernanceActionId <$> mPrevGovActId
+  let prevGovActId = Ledger.maybeToStrictMaybe $ uncurry createPreviousGovernanceActionId <$> mPrevGovernanceActionId
       constitutionAnchor = Ledger.Anchor
         { Ledger.anchorUrl = unConstitutionUrl constitutionUrl
         , Ledger.anchorDataHash = constitutionHash
         }
       govAct = ProposeNewConstitution prevGovActId constitutionAnchor
-      sbe = conwayEraOnwardsToShelleyBasedEra cOn
-      proposalProcedure = createProposalProcedure sbe network deposit stakeKeyHash govAct proposalAnchor
+      sbe = conwayEraOnwardsToShelleyBasedEra eon
+      proposalProcedure = createProposalProcedure sbe networkId deposit stakeKeyHash govAct proposalAnchor
 
   firstExceptT GovernanceActionsCmdWriteFileError . newExceptT
-    $ conwayEraOnwardsConstraints cOn
-    $ writeFileTextEnvelope outFp Nothing proposalProcedure
+    $ conwayEraOnwardsConstraints eon
+    $ writeFileTextEnvelope outFile Nothing proposalProcedure
 
 -- TODO: Conway era - After ledger bump update this function
 -- with the new ledger types
-runGovernanceActionCreateNewCommitteeCmd
-  :: ConwayEraOnwards era
-  -> UpdateCommitteeCmd
+runGovernanceActionCreateNewCommitteeCmd :: ()
+  => GoveranceActionUpdateCommitteeCmdArgs era
   -> ExceptT GovernanceActionsError IO ()
-runGovernanceActionCreateNewCommitteeCmd cOn (UpdateCommitteeCmd network deposit retAddr proposalUrl proposalHashSource old new q prevActId oFp) = do
-  let sbe = conwayEraOnwardsToShelleyBasedEra cOn -- TODO: Conway era - update vote creation related function to take ConwayEraOnwards
-      govActIdentifier = Ledger.maybeToStrictMaybe $ uncurry createPreviousGovernanceActionId <$> prevActId
-      quorumRational = toRational q
+runGovernanceActionCreateNewCommitteeCmd
+    Cmd.GoveranceActionUpdateCommitteeCmdArgs
+      { Cmd.eon
+      , Cmd.networkId
+      , Cmd.deposit
+      , Cmd.returnAddress
+      , Cmd.proposalUrl
+      , Cmd.proposalHashSource
+      , Cmd.oldCommitteeVkeySource
+      , Cmd.newCommitteeVkeySource
+      , Cmd.requiredQuorum
+      , Cmd.mPrevGovernanceActionId
+      , Cmd.outFile
+      } = do
+  let -- TODO: Conway era - update vote creation related function to take ConwayEraOnwards
```

Comment:
This comment looks like it can be removed.

Reply:
Done!